CHANGELOG.md

```diff
@@ -50,7 +50,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where identity column header had incorrect foreground color in the  Dark theme. [#6796](https://github.com/JabRef/jabref/issues/6796)
 
 ### Removed
-
+- We removed the menu entry "Manage external file types" because it's already in 'Preferences' dialog [#6991](https://github.com/JabRef/jabref/issues/6991)
```

Comment:
Please follow [markdown-lint](https://github.com/DavidAnson/markdownlint) - have an empty line above and below an item list - see also the style of the other entries in the file